src/Cache/Engine/RedisEngine.php

```diff
@@ -89,6 +93,23 @@ public function init(array $config = []): bool
 
         parent::init($config);
 
+        $map = [
+            'ssl_ca' => 'cafile',
+            'ssl_key' => 'local_pk',
+            'ssl_cert' => 'local_cert',
+        ];
```

Comment:
I think it is preferable to keep this configuration as a flat map, and generate this `$context` structure when making the client connection.  Currently users *could* modify configuration keys and them not take effect on future operations. I'd prefer we take on more complexity/mess to avoid this issue.